src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/Exceptions/ConnectionAbortedException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions
+{
+    public class ConnectionAbortedException : OperationCanceledException
+    {
+        public ConnectionAbortedException() :
+            this("The connection was aborted")
```

Comment:
Because it's in Core only and we don't cross reference resources from other assemblies. I could add a resource file to transport abstractions but I'm not going to because it's pubternal and not being localized 😝